pkg/services/container/executor.go

```diff
@@ -31,7 +31,12 @@ func NewExecutionService(exec ServiceExecutor) Server {
 }
 
 func (s *executorSvc) Put(ctx context.Context, req *container.PutRequest) (*container.PutResponse, error) {
-	respBody, err := s.exec.Put(ctx, req.GetMetaHeader().GetOrigin().GetSessionToken(), req.GetBody())
+	meta := req.GetMetaHeader()
+	for origin := meta.GetOrigin(); origin != nil; origin = meta.GetOrigin() {
+		meta = origin
+	}
+
+	respBody, err := s.exec.Put(ctx, meta.GetSessionToken(), req.GetBody())
```

Comment:
Why is there #1285 in the commit message?

Reply:
Typo.